### DISCUSSION TITLE
Save default page background "squared" instead of "ruled" paper?

### DISCUSSION THREAD

**Discussion by justxi** [xournalpp/xournalpp#3097] [Q&A]
Hi,

currently when I start xournal++ I see always the ruled paper background.
Is it possible to save squared paper as a default for the next startup?

**Comment by rolandlo** [comment]
Yes, go to menu `Journal > Configure page template`

**Comment by justxi** [comment]
Thanks, but on my system it is not saved for the next start up of xournal++.  Next time I use xournal++ it is has ruled paper as default again.

**Comment by rolandlo** [comment]
That's very strange. I can't reproduce it and don't remember having heard of this behavior, although it may depend on the version of Xournal++ you use. Consider installing [version 1.1.0+dev](https://github.com/xournalpp/xournalpp/releases/tag/nightly).

**Comment by justxi** [comment]
I am using 1.0.20 for windows. 
I will try the dev version.
